pkg/proxy/winkernel/proxier_test.go

```diff
@@ -20,39 +20,72 @@ limitations under the License.
 package winkernel
 
 import (
+	"encoding/json"
 	"fmt"
 	"net"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/Microsoft/hcsshim/hcn"
 	v1 "k8s.io/api/core/v1"
 	discovery "k8s.io/api/discovery/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/kubernetes/pkg/proxy"
 	"k8s.io/kubernetes/pkg/proxy/healthcheck"
+	fakeHcn "k8s.io/kubernetes/pkg/proxy/winkernel/testing"
```

Comment:
```
pkg/proxy/winkernel/proxier_test.go:	fakeHcn "k8s.io/kubernetes/pkg/proxy/winkernel/testing"
!!! Some package aliases break go conventions.
To fix these errors, do not use capitalized or underlined characters
```
/hold

The CI failure should be fixed. Feel free to unhold once this is fixed.

Reply:
Issue with package name convention fixed.

/unhold